### PR TITLE
Port Line to TypeScript

### PR DESCRIPTION
**Pull request by evancharlton**
This includes a number of @ts-ignore directives -- I'm pretty sure that
d3's type annotations aren't accurate since this works anyway. Weird.

### CODE REVIEW

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-298.surge.sh

File: src/components/Points/index.tsx [hunk 1]
```diff
@@ -29,11 +29,11 @@ const Points: React.FunctionComponent<Props> = ({
...
   xAccessor,
-  x0Accessor,
-  x1Accessor,
+  x0Accessor = undefined,
+  x1Accessor = undefined,
   yAccessor,
-  y0Accessor,
-  y1Accessor,
+  y0Accessor = undefined,
+  y1Accessor = undefined,
```
Comment (valerii-cognite) [inline src/components/Points/index.tsx]: is that `undefined` a requirement here?

Comment (evancharlton) [reply src/components/Points/index.tsx]: Good catch; no.

**Comment by cognite-cicd** [comment]
[pr-server]
The storybook for this PR is hosted on https://griff-react-pr-298.surge.sh

**Pull request merged**